<launch>
    <arg name="dof" default="1" />
    <arg name="present_jointstate_topic" default="/motors/present_states" />
    <arg name="goal_jointstate_topic" default="/motors/goal_states" />

    <!-- viz of current robot joint states -->
    <include file="$(find trackerrr)/launch/viz_robot.launch">
        <arg name="dof" value="$(arg dof)" />
    </include>

    <!-- dynamixel sim -->
    <node pkg="trackerrr" type="dynamixelsim_node" name="dynamixelsim" output="screen">
        <param name="dof" value="$(arg dof)" />
        <param name="present_jointstate_topic" value="$(arg present_jointstate_topic)" />
        <param name="goal_jointstate_topic" value="$(arg goal_jointstate_topic)" />
        <param name="p" value="2.0" />
        <param name="i" value="0.5" />
        <param name="d" value="0.1" />
        <param name="vel_max" value="1.5708" />
    </node>

    <node pkg="rviz" type="rviz" name="rviz_dynamixelsimtest_node" args="-d $(find trackerrr)/cfg/rviz.rviz" />
</launch>